### PR TITLE
added coding for voice assistant

### PR DESCRIPTION
**Pull request by adnan832**
javascript 1 week 4 homework

### CODE REVIEW

File: javascript/javascript1/week4/voiceAssistant.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+function getReply(command) 
+{
+
+    if (typeof command !== "string") {
+        return "please enter valid string"
+    }
+    let stringName = command.toLowerCase().split(' '); // This will make all letters into lowercase
+
+    if (command.toLowerCase().includes('hello my name is')) {
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/voiceAssistant.js:17]: You are using `command.toLowerCase()` multiple times so create a new variable for it.
```suggestion
    const lowerCaseCommand = command.toLowerCase();
    if (lowerCaseCommand.includes('hello my name is')) {
```

File: javascript/javascript1/week4/voiceAssistant.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                return `${num1} ${operator} ${num2} = ${num1 + num2}`;
+                break;
+            case "*":
+                return `${num1} ${operator} ${num2} = ${num1 * num2}`;
+                break;
+            case "-":
+                return `${num1} ${operator} ${num2} = ${num1 - num2}`;
+                break;
+            case "/":
+                return `${num1} ${operator} ${num2} = ${(num1 / num2).toFixed(2)}`;
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/voiceAssistant.js:126]: What is this result if `num2 = 0`?

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week4/voiceAssistant.js (hunk 1 above)
Comment (adnan832) [reply javascript/javascript1/week4/voiceAssistant.js:17]: will keep this useful information for future programming

File: javascript/javascript1/week4/voiceAssistant.js (hunk 2 above)
Comment (adnan832) [reply javascript/javascript1/week4/voiceAssistant.js:126]: yeah I have forgotten that case will do it shortly

**Pull request merged**